### ISSUE TITLE
Gmail feed not update

### ISSUE THREAD

**Issue by SkyMaverick** [QuiteRSS/quiterss#1078]
Gmail new inbox-messages feed (Atom) loaded after subscribe (login/pass correct), but not update after this. Previous stable build works fine.
System: Linux Mint 18.3 x64 Cinnamon. QRSS 0.18.11 from official PPA.

**Comment by Reisenfag** [comment]
Google has closed access to unauthorized applications. Try logging in to your account in the internal QuiteRSS browser.

**Comment by SkyMaverick** [comment]
> Google has closed access to unauthorized applications.

Yes, but after the first subscription the feed loaded however (proof. [Imgur](https://imgur.com/LNSI1u9)).

PS. But if authorize with internal browser, feed loading successfully. Thanks for the idea.

**Comment by shravanshetty1** [comment]
How do I login using the internal browser?  cant seem to get this to work

Edit -
Enabling "less secure apps" in google security settings worked for me, would still prefer suggested solution
